boot/storage.txt

```diff
@@ -0,0 +1,424 @@
+    
+    In order for a disk to be used as an installation medium, the storage space 
+    must be formatted to the desired file system type.  The following are 
+    example commands that are useful for creating those partition types. Note 
+    that not all commands will be available on your distribution Live CD.  For 
+    this reason, I would recommend consulting the individuals distribution help 
+    pages.
+    
+    +------------------+-------------------+-----------------------------------+
+    |    Filesystem    |   Command         |   Userspace Utilities (Repo)      |
+    +------------------+-------------------+-----------------------------------+
+    |                  |                   |                                   |
+    |    btrfs         |   mkfs.btrfs      |   btrfs-progs (Community)         |
+    |                  |                   |                                   |
+    |    ext2          |   mkfs.ext2       |   e2fsprogs (Extra)               |
+    |                  |                   |                                   |
+    |    ext3          |   mkfs.ext3       |   e2fsprogs (Extra)               |
+    |                  |                   |                                   |
+    |    ext4          |   mkfs.ext4       |   e2fsprogs (Extra)               |
+    |                  |                   |                                   |
+    |    f2fs          |   mkfs.f2fs       |   f2fs-tools                      |
+    |                  |                   |                                   |
+    |    jfs           |   mkfs.jfs        |   jfsutils                        |
+    |                  |                   |                                   |
+    |    reiserfs      |   mkfs.reiserfs   |   reiserfsprogs                   |
+    |                  |                   |                                   |
+    |    xfs           |   mkfs.xfs        |   xfsprogs                        |
+    |                  |                   |                                   |
+    |    vfat          |   mkfs.vfat       |   dosfstools                      |
+    |                  |                   |                                   |
+    |    NTFS          |   mkfs.ntfs       |   ntfs-3g (Community)             |
+    |                  |                   |                                   |
+    |    swap          |   mkswap          |                                   |
+    |                  |                   |                                   |    
+    +------------------+-------------------+-----------------------------------+
+
+
+    [3.3] Swap File
+    ____________________________________________________________________________    
+    
+    As an alternative to creating an entire partition, a swap file offers the 
+    ability to vary its size on-the-fly, and is more easily removed altogether. 
+    This may be especially desirable if disk space is at a premium.
```

Comment:
Are there any downsides to a swapfile? We list only benefits. If true... why include swap partitions at all?

Reply:
To my knowledge, after the rise in popularity of solid state drives, there are basically no downsides. *Unless* you're on a spinning disk. Then a swap partition is almost certainly better (and that's solely for access times).

The biggest nonhardware-related downside is basically a problem that plagues... Files in general. We saw a recent wave of posts like these[1] a while back, for instance. 


[1] https://www.reddit.com/r/archlinux/comments/h127wc/swapfile_error_on_boot_after_57_update/?utm_medium=android_app&utm_source=share